### PR TITLE
Tonia design fix

### PR DESCRIPTION
**Pull request by tone4ka**
bad adaptive, needs a design fix

### CODE REVIEW

**Comment by Srr233** [comment]
What the commits are? Why names are like that?
image

**Comment by tone4ka** [comment]
I'm sorry. I will create correct names next time

File: style.css [hunk 1]
```diff
@@ -49,7 +48,7 @@ input {
     border: none;
 }
 input, select, button {
-    font-size: 1rem;
+    font-size: 3vh;
```
Comment (Srr233) [inline style.css]: Why do you use vh for font-size?

**Comment by Srr233** [comment]
1. Some problems with stars, it doesn't appear for start word.
2. Fix design (as we were talking) :)

**Pull request closed without merging**